packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/attributedstring/AttributedString.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.views.text.attributedstring
+
+/** Interface for an attributed string */
```

Comment:
This comment, and same for `AttributedStringFragment` are not meaningful to someone who wants to learn what this interface represents and is used for.